config/settings.yml

```diff
@@ -366,9 +366,8 @@ apiv3_cors_origins:
 apiv3_docs_enabled:
   default: true
   format: boolean
+apiv3_max_page_size:
+  default: 500
```

Comment:
I'd increase it to a 1000 as a default to reduce the times users run into problems.